providers/implementations/ciphers/cipher_aes_gcm_hw_ppc.inc

```diff
@@ -109,25 +91,21 @@ static size_t aes_p10_gcm_crypt(const unsigned char *in, unsigned char *out, siz
 }
 
 size_t ppc_aes_gcm_encrypt_wrap(const unsigned char *in, unsigned char *out, size_t len,
-                         const void *key, unsigned char ivec[16], u64 *Xi)
+                                const void *key, unsigned char ivec[16], u64 *Xi)
 {
-    size_t n;
     int encrypt;
 
     encrypt = 1;
```

Comment:
You could even drop this variable and pass the number 1 directly. And likewise below in decrypt.

Reply:
fixed.